Dockerfile

```diff
@@ -39,6 +39,16 @@ ENV GOPATH /go
 ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
 
+# Install Crystal 0.31
```

Comment:
Docker file is in separate repo -- https://github.com/ReformedDevs/challenge-docker

Reply:
Created a pull request at that repo with updated Dockerfile w/ Crystal support.